src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ASN1.cs

```diff
@@ -127,4 +127,12 @@ internal class SafeSharedAsn1ObjectHandle : SafeInteriorHandle
         {
         }
     }
+
+    internal class SafeSharedAsn1OctetStringHandle: SafeInteriorHandle
```

Comment:
Yes, even though they have the same functionality, the idea was to keep the API and internal handle passing type safe. I also feel this makes the code far more readable as I know what handle to expect from the interop methods.